src/pages/Home.vue

```diff
@@ -108,12 +105,12 @@ export default {
   computed: {
     /** Whether authentication method chosen is an ID entry */
     isAuthTypeID() {
```

Comment:
That's what is done below. But the reason I have two different methods for `ID` and `OTP` auth type is because they have different components to render.

Reply:
Should be combined, yo. Can you think of a way to combine them? Components can be part of a return data structure. 

It doesn't have to be in this PR, but make a note and after this ships, we can prioritize it, because this will get messier with time

Reply:
Made an issue.